hcl2template/types.build.hcp_packer_registry.go

```diff
@@ -48,7 +48,8 @@ func (p *Parser) decodeHCPRegistry(block *hcl.Block, cfg *PackerConfig) (*HCPPac
 		BuildLabels  map[string]string `hcl:"build_labels,optional"`
 		Config       hcl.Body          `hcl:",remain"`
 	}
-	diags := gohcl.DecodeBody(body, cfg.EvalContext(LocalContext, nil), &b)
+	ectx := cfg.EvalContext(DatasourceContext, nil)
```

Comment:
I think it makes more sense if this is `BuildContext` considering that the `hcp_packer_registry` is inside the build block. What do you think?

Reply:
Good point. I'll update.